ecs-cli/modules/compose/cli/ecs/app/service_command.go

```diff
@@ -136,3 +136,34 @@ func deploymentConfigFlags(specifyDefaults bool) []cli.Flag {
 		},
 	}
 }
+
+func loadBalancerFlags() []cli.Flag {
+	targetGroupArnUsageString := "[Optional] The full Amazon Resource Name (ARN) of the Elastic Load Balancing target group associated with a service."
+	containerNameUsageString := "[Optional] Mandatory if target-group-arn is specified. The container name (as it appears in a container definition)."
```

Comment:
Would you mind changing this to:

"[Optional] Specifies the container name (as it appears in a container definition). This parameter is required if a load balancer or target group is specified."